Doc/library/heapq.rst

```diff
@@ -82,6 +82,59 @@ The following functions are provided:
    on the heap.
 
 
+For max heaps, the reverse of a heap, the following functions are provided:
```

Comment:
I wouldn't say the "reverse" but rather define what a max heap in the top paragraph where we define what a heap is. Currently:

> Heaps are binary trees for which every parent node has a value less than or
equal to any of its children. We refer to this condition as the heap invariant.

We can say:

```rst
Min-heaps (resp. max-heaps) are binary trees for which every parent node
has a value less than (resp. greater than) or equal to any of its children.
We refer to this condition as the heap invariant. Unless stated otherwise,
*heaps* refer to min-heaps.
```